torchaudio/prototype/models/squim/objective.py

```diff
@@ -0,0 +1,289 @@
+import math
+from typing import List, Optional, Tuple
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+
+def transform_wb_pesq_range(x: float) -> float:
+    """Map from the P.862 score to the P.862.2 score.
+    Officially, only the metric defined by ITU-T P.862 is called 'PESQ score',
```

Comment:
Change in the language of the comment. 

The metric defined by ITU-T P.862 is called often called 'PESQ score', and is defined for narrow-band signals and has a value range of [-0.5, 4.5] exactly. Here, we are using the metric defined by ITU-T P.862.2, commonly known as 'wide-band PESQ' and will be referred to as "PESQ score".

Reply:
addressed in [e906bba](https://github.com/pytorch/audio/pull/3042/commits/e906bba161e67c2917a703b11f26e54707f33b93)